ote/test/clj/ote/transit_changes/detection_window_test.clj

```diff
@@ -0,0 +1,55 @@
+(ns ote.transit-changes.detection-window-test
```

Comment:
"design for testability" i.e. should the tested functions take the detection window value as an argument.
That way tests could test using different values...in the other hand would that bring any value?